config/routes.rb

```diff
@@ -1,6 +1,18 @@
 Rails.application.routes.draw do
+  resources :template_syncs, :only => [:index]
+
+  resources :ui_template_syncs, :only => [] do
+    collection do
+      get 'sync_settings'
+      post 'import'
+      post 'export'
+    end
+  end
+
+  match '/template_syncs/*page' => 'template_syncs#index', :via => [:get]
```

Comment:
I know I am a bit late for the party,
but would you like to use react router now that it has been added ? :)

Reply:
I would like that very much :), one problem is that we inject quite a lot of things into props when we `mount_react_component` in view which is sort of "context" for the plugin and I am not sure if I can make it work in a sane way with what we currently have in core. I will look into it though.